### PR TITLE
WER unittest fix

### PR DESCRIPTION
**Pull request by okuchaiev**
This PR only fixes ASR WER metric unittest

### CODE REVIEW

File: tests/collections/asr/test_asr_metrics.py [hunk 1]
```diff
@@ -73,56 +104,26 @@ def test_wer_function(self):
...
-                    targets=self.__string_to_tensor(reference),
-                    target_lengths=torch.tensor([len(reference)]),
-                )
-                .detach()
-                .cpu()
-            )
-            return res[0] / res[1]
-
         for test_id in range(256):
             n1 = random.randint(0, 512)
```
Comment (blisc) [inline tests/collections/asr/test_asr_metrics.py:124]: Please change to randint(1, 512) when both n1 and n2 are zero you get nan != Inf error

Comment (okuchaiev) [reply tests/collections/asr/test_asr_metrics.py]: fixed. Thanks!
